### PR TITLE
Modifytest config

### PR DESCRIPTION
**Pull request by almmechanics**
Update `test` config to be passive wrt github and not process any repos
Added missing `sauce` api key for the build

### CODE REVIEW

File: cac-test.yml [hunk 1]
```diff
@@ -138,7 +138,13 @@ unclassified:
 #              id: slack-token
 #              secret: '${SLACK_TOKEN}'
 #              description: Slack token
+          - sauce:
```
Comment (timja) [inline cac-test.yml]: this shouldn't be needed

File: cac-test.yml [hunk 2]
```diff
@@ -138,7 +138,13 @@ unclassified:
...
+              apiKey: "${sauce-access-key}"
+              description: "Sauce Labs Reform Key"
+              id: "reform_tunnel"
+              scope: GLOBAL
+              username: "reform"
 
 jobs:
```
Comment (timja) [inline cac-test.yml]: maybe comment this out too

File: cac-test.yml (hunk 1 above)
Comment (almmechanics) [reply cac-test.yml]: code removed for sauce

File: cac-test.yml (hunk 2 above)
Comment (almmechanics) [reply cac-test.yml]: commented out `jobs:` section

File: cac-test.yml [hunk 3]
```diff
@@ -123,10 +123,10 @@ unclassified:
   #       serverAuthenticationToken: '${SONARCLOUD}'
   #       serverUrl: 'https://sonarcloud.io'
 
-#credentials:
-#  system:
-#    domainCredentials:
-#      - credentials:
+credentials:
```
Comment (timja) [inline cac-test.yml]: probably don't need this change here

**Pull request merged**